yb_migrate/cmd/exportData.go

```diff
@@ -191,12 +193,20 @@ func checkTableListFlag(tableListString string) {
 func checkDataDirs() {
 	exportDataDir := exportDir + "/data"
 	flagFilePath := exportDir + "/metainfo/flags/exportDataDone"
+	dfdFilePath := exportDir + datafile.DESCRIPTOR_PATH
 	if startClean {
+		proceed := utils.AskPrompt(
+			"CAUTION: Using --start-clean will erase any exported data files. Do you want to proceed")
```

Comment:
Let's drop this prompt.

Reply:
@amit-yb i added for startClean to be uniform for export schema and data, any specific reason to drop this?